### PR TITLE
Hook up the edit widget button

### PR DESCRIPTION
**Pull request by Kegsay**
Along with a few minor fixes:
 - Use the right translation string in en_EN.json
 - Fix typos in comments

### CODE REVIEW

File: src/components/views/elements/AppTile.js [hunk 1]
```diff
@@ -56,6 +59,7 @@ export default React.createClass({
     },
 
     componentWillMount: function() {
+        this._onEditClick = this._onEditClick.bind(this);
```
Comment (dbkr) [inline src/components/views/elements/AppTile.js]: This uses createClass so you shouldn't need to bind this

File: src/components/views/elements/AppTile.js [hunk 2]
```diff
@@ -86,8 +90,17 @@ export default React.createClass({
         });
     },
 
-    _onEditClick: function() {
-        console.log("Edit widget %s", this.props.id);
+    _onEditClick: function(e) {
+        console.log("Edit widget ID ", this.props.id);
+        if (e) {
```
Comment (dbkr) [inline src/components/views/elements/AppTile.js]: You'll always have an event if it's given to an onClick handler, so this probably isn't strictly necessary.

Edit: actually come to think of it, what default are you preventing anyway?

Comment (Kegsay) [inline src/components/views/elements/AppTile.js]: I don't think there is any default tbh - I was just doing what the `AppsDrawer` does for `onClickAddWidget`. Will remove.

File: src/components/views/elements/AppTile.js (hunk 1 above)
Comment (Kegsay) [inline src/components/views/elements/AppTile.js]: Yeah, I thought so too, but was wrangling issues where `this._scalarClient` was not defined and I was clutching at straws to see if it was a bind problem (it was due to something else). I should've removed this before committing, thanks.

File: src/components/views/elements/AppTile.js (hunk 2 above)
Comment (Kegsay) [inline src/components/views/elements/AppTile.js]: Done. https://github.com/matrix-org/matrix-react-sdk/pull/1218/commits/ae3e7a7627b9331107f547bd58f7e222d1262d6c

File: src/components/views/elements/AppTile.js (hunk 1 above)
Comment (Kegsay) [inline src/components/views/elements/AppTile.js]: Done. https://github.com/matrix-org/matrix-react-sdk/pull/1218/commits/ae3e7a7627b9331107f547bd58f7e222d1262d6c

**Comment by rxl881** [comment]
Great. LGTM. Thanks @Kegsay

**Pull request merged**